### DISCUSSION TITLE
Why are these Common Error settings not catching theses lines with single dialogue markers?

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5279] [General]
This is driving me nuts. Yes, I checked to make sure they're regular dashes.

![image](https://user-images.githubusercontent.com/82276985/130704582-bb8a1741-20e9-4698-b6a9-450e4dc4f25a.png)

**Comment by niksedk** [comment]
Try the "Fix dialogs via dialog style: ...."

**Reply by DrReddP** [reply]
That worked nicely, thank you
